### ISSUE TITLE
Behavior of Date Picker

### ISSUE THREAD

**Issue by GeorgeTaveras1231** [xogroup/union#298]
The `Field` + `DatePicker` as a dropdown combination is currently implemented in a way that would be disruptive for a user because it pushes content down when opened. This could be addressed by consumers by adding a `position: absolute` container. Should the date picker support a `isDropdown` feature to support this behavior?

![datepicker](https://user-images.githubusercontent.com/3114629/29372733-f7aa39e4-8279-11e7-8bac-69498cce389f.gif)

**Comment by hsbacot** [comment]
This would just be for the demo page, the consumer should be able to do whatever they need by implementing their own container / field combo

**Comment by GeorgeTaveras1231** [comment]
@hsbacot I think this will be a very common pattern though, wouldn't it be nice to have a centralized implementation for that use case?

**Comment by marioharper** [comment]
@GeorgeTaveras1231 yeah I kind of agree that there should be a `Datepicker` implementation with `Field` since I think it is our only current use case for having a date picker.
